services/api-server/src/simcore_service_api_server/api/routes/files.py

```diff
@@ -59,12 +62,12 @@
 #
 #
 
-_COMMON_ERROR_RESPONSES: Final[dict] = {
+_FILE_STATUS_CODES: dict[int | str, dict[str, Any]] = {
```

Comment:
MINOR: I find  `_COMMON_ERROR_RESPONSES` more uniform (i.e. all routes files can have one of those) and informative (refers to `responses` parameter)

Reply:
OK, fair enough. But I already thinks that it is quite clear that this thing refers to the status codes which can be returned. But I see your point that I could have added a `_RESPONSE` after the name